example/feature_demo/demo_service.pb.gorm.go

```diff
@@ -1000,10 +1000,37 @@ type IntPointServiceIntPointWithAfterList interface {
 
 // ListSomething ...
 func (m *IntPointServiceDefaultServer) ListSomething(ctx context.Context, in *google_protobuf2.Empty) (*ListSomethingResponse, error) {
-	out := &ListSomethingResponse{}
+	db := m.DB
+	if custom, ok := interface{}(in).(IntPointServiceSomethingWithBeforeListSomething); ok {
+		var err error
+		if db, err = custom.BeforeListSomething(ctx, db); err != nil {
+			return nil, err
+		}
+	}
+	res, err := DefaultListSomething(ctx, db)
```

Comment:
I don't think that this function exists, which means that the way this call is generated would have to be fixed.

Reply:
I might be missing something but I think this function does exist.
I can see the rpc defined in  feature_demo/demo_service.proto
And the function is generated in /feature_demo/demo_service.pb.gorm.go (line 642)
It looks like this function existed before my changes just was never used

Reply:
Hi @Calebjh any thoughts on the above?